crates/common/trie/trie.rs

```diff
@@ -165,7 +165,7 @@ impl Trie {
         Ok(())
     }
 
-    pub fn commit_vec_aka_state_diff(&mut self) -> Vec<(NodeHash, Vec<u8>)> {
+    pub fn commit_without_storing(&mut self) -> Vec<(NodeHash, Vec<u8>)> {
```

Comment:
can you document this function adding what the return types are? (specially the Vec<u8> taht is not completely clear what it is.